Jellyfin.Data/Entities/BaseItemTrailerType.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Jellyfin.Data.Entities;
+
+/// <summary>
+/// Enum TrailerTypes.
+/// </summary>
+public class BaseItemTrailerType
+{
+    /// <summary>
+    /// Gets or Sets Numerical ID of this enumeratable.
+    /// </summary>
+    public required int Id { get; set; }
```

Comment:
Same as above

Reply:
Is the int representation of `TrailerType` enum